testnet/tools/nns-tools/lib/proposals.sh

```diff
@@ -146,29 +196,51 @@ generate_sns_bless_wasm_proposal_text() {
     RELATIVE_CODE_LOCATION="$(echo "$CANISTER_CODE_LOCATION" | sed "s/$ESCAPED_IC_REPO/./g")"
 
     OUTPUT=$(
-        cat <<EOF
-## Proposal to Publish the SNS $CAPITALIZED_CANISTER_TYPE Canister WASM to SNS-W
-### Proposer: DFINITY Foundation
-### Canister Type: $CANISTER_TYPE
-### Git Hash: $NEXT_COMMIT
-### New Wasm Hash: $WASM_SHA
----
-## Features
-TODO ADD FEATURE NOTES
-## Release Notes
+        cat <<++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+# Publish SNS $CAPITALIZED_CANISTER_TYPE Upgrade to Commit $SHORT_NEXT_COMMIT
```

Comment:
I think  it should be "Publish... Upgrade _at_ Commit ..."

Reply:
"Upgrade at" sounds weird. I changed this to "Publish SNS Governance WASM Built at Commit $COMMIT_ID".